src/test/java/org/apache/commons/lang3/time/StopWatchTest.java

```diff
@@ -448,4 +470,712 @@ public void testToStringWithMessage() throws InterruptedException {
         final String splitStr = watch.toString();
         assertEquals(12 + MESSAGE.length() + 1, splitStr.length(), "Formatted split string not the correct length");
     }
+
+    @Nested
+    public class SupplierSupportRelatedTests {
+        @Test
+        public void testGet() {
+            final StopWatch watch = StopWatch.create();
+
+            assertTrue(watch.isStopped());
+
+            Supplier<String> supplier = () -> "Foobar";
+            String result = watch.get(supplier);
+
+            assertEquals("Foobar", result);
+            assertTrue(watch.isSuspended());
+        }
+
+        @Test
+        public void testGetWhenStopWatchHasBeenSuspended() throws Throwable {
+            final StopWatch watch = StopWatch.create();
+
+            watch.start();
+            watch.suspend();
+
+            assertTrue(watch.isSuspended());
+
+            Supplier<String> supplier = () -> "Foobar";
+            String result = watch.get(supplier);
```

Comment:
`result` local variable is not necessary.

Reply:
Removed all local variables for single line invocations, but left then for multline statements